opentracing-api/pom.xml

```diff
@@ -25,8 +25,8 @@
         <version>0.1.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>opentracing</artifactId>
-    <name>OpenTracing</name>
+    <artifactId>opentracing-api</artifactId>
```

Comment:
Yes yes, I just meant in the dep management file...

Given that, I would suggest we keep this as `opentracing` since that is the by-far-most-important-case. Fine with me if we eventually have `opentracing-basictracer` or `opentracing-spi` (etc etc) in addition.

Not a strongly-held opinion in any case. @michaelsembwever thoughts?